### PR TITLE
subDAO Onboarding Fees HIP Draft

### PR DESCRIPTION
**Pull request by ferebee**

### CODE REVIEW

**Comment by samgutentag** [comment]
Dang, great work! I was going to make suggestions inline, but it would get messy quickly. instead, see [this gist of some edits](https://gist.github.com/samgutentag/3deba0e24b985fb8eaa5c6c3c7454650)

Proposed edits:
- capitalize "Hotspot"
- swap LoRa for IOT
- swap 5G for MOBILE

The swap to use the subDAO name rather than specific wireless technologies is intended to conform to the existing network and subDAO language and accommodate future hardware to preserve the longevity of this HIP.

**Comment by ferebee** [comment]
Thanks, looks great! I think I made all the changes you suggested.

**Review by vincenzospaghetti — CHANGES_REQUESTED** [review]

File: XXXX-subdao-onboarding-fees.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+However, current MOBILE Hotspots, as introduced by FreedomFi, contain both an IOT Hotspot and a MOBILE Hotspot. While HIP-53 specifies that MOBILE Hotspots should burn a $40 onboarding fee, just like IOT Hotspots, the existing implementation only onboarded the IOT network component. All active MOBILE Hotspots have been onboarded to the IOT network, but no separate onboarding process for the MOBILE network was ever implemented.
+
+As a result, no onboarding fees have yet been burned towards the MOBILE subDAO.
+
+This prevents the tokenomics based on DAO Utility Score, as defined in HIP-51, from working as designed.
+
+HIP-XX solves this problem by requiring that Hotspots participating in a particular Helium subDAO must burn an onboarding fee towards that subDAO. Specific fee schedules will be determined by subDAO governance. A mechanism is specified whereby the existing MOBILE Hotspots may be onboarded retroactively.
+
```
[the remark this answers is not in the document]
Comment (ferebee) [reply XXXX-subdao-onboarding-fees.md:22]: added that

File: XXXX-subdao-onboarding-fees.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+As the Helium DAO grows, new devices may be introduced that participate in multiple current and future subDAOs at once. Both the IOT and the MOBILE subDAOs will benefit if appropriate onboarding fees are paid to all subDAOs involved.
+
+# Stakeholders
+
+This proposal is relevant to:
+
+- Existing and future IOT and MOBILE Hotspot owners, who are affected by the clarification of fees and the MOBILE DAO Utility Score.
+- MOBILE Hotspot manufacturers, who may need to budget for onboarding fees as decided by the MOBILE subDAO.
+- Anyone designing a future subDAO.
+
```
[the remark this answers is not in the document]
Comment (ferebee) [reply XXXX-subdao-onboarding-fees.md:38]: added the first bit, think the second is a bit farfetched

File: XXXX-subdao-onboarding-fees.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+In the future, devices may be introduced that participate in many different subDAOs simultaneously, such as IOT, MOBILE, future networks such as Wi-Fi, and other services such as dVPN or storage networks. In some cases, it may become possible to onboard existing Hotspots to new subDAOs, through software or hardware upgrades.
+
+To ensure that all participating Hotspots contribute appropriately to the various DAO Utility Scores, we propose the following provisions.
+
+# Provisions
+
+## Independent Onboarding
+
+If a Hotspot participates in one or more Helium subDAOs, it must be onboarded to each subDAO individually, and the onboarding fee in DC as set by each subDAO must be accordingly burned separately towards that subDAO.
+
```
[the remark this answers is not in the document]
Comment (ferebee) [reply XXXX-subdao-onboarding-fees.md:66]: can’t give a $ value, it depends on subDAO governance in the general case

File: XXXX-subdao-onboarding-fees.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+## DAO Utility Score
+
+As specified in HIP-51, each Hotspot contributes to the DAO Utility Score of its subDAO according to its onboarding fee. For the purposes of this calculation, Hotspots which participate in multiple subDAOs shall be considered as multiple independent Hotspots, one for each subDAO, and each shall contribute separately to the DAO Utility Score of its subDAO with the onboarding fee burned towards that subDAO. No onboarding fee shall be double-counted towards multiple subDAOs.
+
+## Fees Are Decided By subDAOs
+
+Each subDAO may set a schedule of onboarding fees for its participating Hotspots through subDAO governance. All onboarding fees must be paid in DC burn from HNT according to procedures specified by the Helium DAO.
+
+Subject to change through subDAO governance by the MOBILE subDAO, the onboarding fee per MOBILE Hotspot to the MOBILE subDAO is $40 as defined in HIP-53, independent of other subDAO onboarding fees paid by the same Hotspot, such as the $40 IOT network onboarding fee that has been paid by the IOT component of existing MOBILE Hotspots under the current implementation.
+
```
[the remark this answers is not in the document]
Comment (ferebee) [reply XXXX-subdao-onboarding-fees.md:76]: Correct, I added it to the Summary.

File: XXXX-subdao-onboarding-fees.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+## Paying the Missing MOBILE Onboarding Fees
+
+In cooperation with Hotspot manufacturers, Helium Foundation may perform one or several one-time burn operations which cover, in total, the $40 onboarding fee for each MOBILE Hotspot which has been onboarded to the IOT network under the existing implementation, and thus neglected to pay for onboarding to the MOBILE subDAO. Once these fees are paid, a corresponding number of MOBILE Hotspots shall be considered, for the purpose of the DAO Utility Score, to be onboarded to the MOBILE network.
```
[the remark this answers is not in the document]
Comment (ferebee) [reply XXXX-subdao-onboarding-fees.md:79]: To clarify the intent, I suggest that Foundation perform the actual burn operation, acting as a gatekeeper. The necessary funds may be contributed by manufacturers or other entities. We trust Foundation to burn the correct amount as specified here, i. e., ca. 3,963 or so MOBILE hotspots x $40. We’ll change this if Foundation (or somebody else) comes up with a different way of resolving the missing fees.

**Comment by vincenzospaghetti** [comment]
one more thing @ferebee - need to add Success Metrics section:

## Success Metrics
What metrics can be used to measure the success of this design? Are any new ETL reports needed to measure the success?

What should we measure to prove a performance increase?
What should we measure to prove an improvement in stability?
What should we measure to prove a reduction in complexity?
What should we measure to prove an acceptance of this by its users?

**Review by vincenzospaghetti — APPROVED** [review]

**Pull request merged**